src/resources/billing.js

```diff
@@ -0,0 +1,100 @@
+import T from 'proptypes';
+
+export const propTypes = {
+  id: T.string,
+  disabled_at: T.string,
+  object: T.string,
+  name: T.string,
+  last4: T.string,
+  expiration_month: T.string,
+  expiration_year: T.string,
+  brand: T.string,
+  country: T.string,
+  verified: T.bool,
+  bank_account: T.string,
+};
```

Comment:
Issue: These should really be props of a `PaymentMethod` because there is no "billing" object

Reply:
added